### DISCUSSION TITLE
在app.tsx中使用antd相关组件

### DISCUSSION THREAD

**Discussion by Jixiangup** [umijs/umi#12336] [Q&A]
![image](https://github.com/umijs/umi/assets/75556346/26227433-ec0f-4591-9cc2-26e879bb71cb)
如图我在app.ts配置request的时候 使用国际化会直接报错 如包括使用antd不在App组件包裹之内也会有waring 应该怎么解决呢？

**Comment by fz6m** [comment]
用全局的 intl 实例上的方法获取吧：

```ts
  const intl = getIntl()
  const msg = intl.formatMessage({ id: 'site.title' })
```
